TRANSLATION.md

```diff
@@ -99,8 +100,8 @@ try {
 
 Als een externe link verwijst naar een artikel in een referentie zoals [MDN] of [Wikipedia], en er bestaat een versie van dat artikel in het Nederlands van degelijke kwaliteit, overweeg dan om in de plaats naar die versie te linken.
 
-[MDN]: https://developer.mozilla.org/en-US/
-[Wikipedia]: https://en.wikipedia.org/wiki/Main_Page
+[mdn]: https://developer.mozilla.org/en-US/
```

Comment:
Waarom zijn hier en hieronder de termen naar lowercase veranderd?

Reply:
Geen idee, heb daar niet aan aangepast. Ook die single quotes niet. Ik zie die hier gewoon nog als dubbele... ff kijken wat er is gebeurd...